transfers/transfer.py

```diff
@@ -16,6 +16,16 @@
 import subprocess
 import sys
 import time
+import ConfigParser
+
+# Read config file
+automationToolsConfigFilePath = '/etc/archivematica/automation-tools/automation-tools.conf'
+config = ConfigParser.SafeConfigParser()
+config.read(automationToolsConfigFilePath)
+logDirectory = config.get('automation-tools', 'logDirectory')
+databaseDirectory = config.get('automation-tools', 'databaseDirectory')
+pidFileDirectory = config.get('automation-tools', 'pidFileDirectory')
```

Comment:
Do we want to specify a default value fo these if it's not found in the config file?